### PR TITLE
feat(blog): add Camunda Modeler v3.5.0 release blog post

### PR DESCRIPTION
**Pull request by philippfromme**

### CODE REVIEW

File: content/post/2019/11/camunda-modeler-3.5.0-released.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
++++
+author = "Philipp Fromme"
+categories = ["Modeling"]
+tags = ["BPMN 2.0", "Tooling", "Release Note"]
+date = "2019-11-27T12:00:00+02:00"
+title = "Camunda Modeler 3.5 Released"
++++
+
+We are happy to announce the release of Camunda Modeler v3.5.0. This release comes with an improved deployment feature and many improvements to the BPMN and DMN modeling experience.
```
Comment (oguzeroglu) [inline content/post/2019/11/camunda-modeler-3.5.0-released.md]: It's not that I'm a native speaker or anything :D However I'd say:

> comes with an improved deployment feature and many **other** improvements to the...

I'm not exactly sure though.

File: content/post/2019/11/camunda-modeler-3.5.0-released.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+We are happy to announce the release of Camunda Modeler v3.5.0. This release comes with an improved deployment feature and many improvements to the BPMN and DMN modeling experience.
+
+[Download the latest release](https://camunda.com/download/modeler/) and start modeling.
+
+<!--more-->
+
+## Remember Authentication When Deploying
+
+To make repeated deployments easier, you can now save your authentication when deploying. The next time you're deploying, you don't have to enter it again.
```
Comment (oguzeroglu) [inline content/post/2019/11/camunda-modeler-3.5.0-released.md]: IMO save your **authentication credentials** sounds better than **save your authentication**

Comment (pinussilvestrus) [reply content/post/2019/11/camunda-modeler-3.5.0-released.md]: maybe `authentication details` ?

Comment (oguzeroglu) [reply content/post/2019/11/camunda-modeler-3.5.0-released.md]: Yep actually sounds better as I guess we save more than just username and password but URL etc. as well.

**Comment by pinussilvestrus** [comment]
image

Do we need that big shadows? And maybe we should remove the ⏸from the right bottom corner.

File: content/post/2019/11/camunda-modeler-3.5.0-released.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+To better align the Camunda Modeler with this best practice as of this release sub-processes cannot be collapsed anymore.
+
+{{< figure class="no-border teaser" src="subprocess.gif" alt="Sub-Processes cannot be collapsed anymore" caption="Sub-Processes cannot be collapsed anymore" >}}
+
+Looking forward we believe that this change will make it easier to model processes that are ready for execution. Let us know what you think about this change in [our forum](https://forum.camunda.org/).
+
+## Connect From/To Anywhere
+
+As of this release, you're able to connect elements regardless of what the direction of the resulting connection will be. If the elements can be connected, you're able to connect them. The same also applies to reconnecting connections to other elements.
```
Comment (pinussilvestrus) [inline content/post/2019/11/camunda-modeler-3.5.0-released.md]: > If the elements can be connected, you're able to connect them.

Sound a little bit too obvious ... Maybe

> If the elements can be connected, the connection will be rendered in the correct direction.

Comment (pinussilvestrus) [reply content/post/2019/11/camunda-modeler-3.5.0-released.md]: The word `connect` is also very over-used in these three sentences. But I can't really think about good synonyms here...

File: content/post/2019/11/camunda-modeler-3.5.0-released.md (hunk 2 above)
Comment (barmac) [reply content/post/2019/11/camunda-modeler-3.5.0-released.md]: Actually the feature is about saving your password or token.

**Comment by oguzeroglu** [comment]
LGTM now

File: content/post/2019/11/camunda-modeler-3.5.0-released.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+This release comes with a couple of improvements to the DMN modeling experience. For more details, check out the [changelog](https://github.com/camunda/camunda-modeler/blob/develop/CHANGELOG.md#dmn).
+
+{{< figure class="no-border teaser" src="dmn.gif" alt="Improved DMN experience" caption="Improved DMN experience" >}}
+
+## Looking Forward
+
+Read the [changelog](https://github.com/camunda/camunda-modeler/blob/master/CHANGELOG.md#340) to learn about all features, bug fixes and numerous additional improvements that made it into this release. [Download the new modeler](https://camunda.com/download/modeler/), give it a try, and, most importantly, give us feedback.
+
+Did we miss anything? Did you spot a bug or would you like to suggest an improvement? Reach out to us via [our forums](https://forum.camunda.org/c/modeler), tweet us [@Camunda](https://twitter.com/Camunda) or file issues you found in the [Camunda Modeler issue tracker](https://github.com/camunda/camunda-modeler/issues/new choose).
```
Comment (barmac) [inline content/post/2019/11/camunda-modeler-3.5.0-released.md]:
```suggestion
Did we miss anything? Did you spot a bug or would you like to suggest an improvement? Reach out to us via [our forums](https://forum.camunda.org/c/modeler), tweet us [@Camunda](https://twitter.com/Camunda) or file issues you found in the [Camunda Modeler issue tracker](https://github.com/camunda/camunda-modeler/issues/new).
```
the link is broken

File: content/post/2019/11/camunda-modeler-3.5.0-released.md (hunk 3 above)
Comment (oguzeroglu) [reply content/post/2019/11/camunda-modeler-3.5.0-released.md]: > As of this release, Camunda Modeler automatically takes care of element connections regardless of what the direction of the resulting connection would be. It means that if a certain connection can be established between different elements, users do not need to worry about the direction semantics, Camunda Modeler will make sure the connection is valid based on the BPM standards. The same logic applies to reconnecting flows to other elements. See the GIF below

How about this?

**Pull request merged**